src/Security/test/Authentication.JwtBearer.Test/JwtBearerAuthenticationBuilderExtensionsTest.cs

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+using FluentAssertions;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+
+namespace Steeltoe.Security.Authentication.JwtBearer.Test;
+
+public sealed class JwtBearerAuthenticationBuilderExtensionsTest
+{
+    [Fact]
+    public void ConfigureJwtBearerForCloudFoundry_AddsExpectedRegistrations()
+    {
+        AuthenticationBuilder serviceCollection = new ServiceCollection().AddAuthentication().AddJwtBearer();
```

Comment:
`AuthenticationBuilder serviceCollection = ...`
The type has changed, please update the variable name.